libmamba/src/api/list.cpp

```diff
@@ -218,7 +219,11 @@ namespace mamba
                 {
                     if (options.canonical)
                     {
-                        LOG_WARNING << "Option --canonical ignored because of --explicit";
+                        LOG_WARNING << "Option --canonical ignored because of --explicit \n";
```

Comment:
`LOG_*` add new lines implicitly.

```suggestion
                        LOG_WARNING << "Option --canonical ignored because of --explicit";
```

Reply:
Nitpick: this message might be slightly more explicit.
```suggestion
                        LOG_WARNING << "Option --canonical ignored because --explicit  was provided";
```